cmake/licenses.cmake

```diff
@@ -0,0 +1,10 @@
+configure_file(${CMAKE_SOURCE_DIR}/LICENSE.md                                                  licenses/sysio/LICENSE.md        COPYONLY)
+
+# TODO: prometheus-cpp, softfloat and CLI11 license files are no longer available since moving to vcpkg
```

Comment:
Should we just add them directly to the repo?

Reply:
Nope, we can copy then out of the the vcpkg installed locations, just not a now thing.  We can get to it prior to launch, its in my notes